### ISSUE TITLE
Add How to create a child theme in the Block theme handbook

### ISSUE THREAD

**Issue by TeBenachi** [WordPress/Documentation-Issue-Tracker#105]
Currently there is no information regarding how to create a child theme for the block theme. I think it would be helpful to add how to create a child theme in the Block theme handbook. 

https://developer.wordpress.org/block-editor/how-to-guides/themes/

**Comment by carolinan** [comment]
Creating a child theme works in the same way for both types of themes. I think we should have it on the existing page.

**Comment by TeBenachi** [comment]
Good point! I think the [Create a Block](https://developer.wordpress.org/block-editor/how-to-guides/themes/create-block-theme/#custom-templates) page would be a good place, possibly after [Example themes](https://developer.wordpress.org/block-editor/how-to-guides/themes/create-block-theme/#example-themes) section. 

**Suggested text:**  

(1) Create a new folder for your child theme in `/wp-content/fse-tutorial-child/`
In this example, we are creating a child theme for the `fse-tutorial` theme.

(2) Create a Style.css as the instruction above, but make sure to indicate it is a child theme. For example

Theme Name: FSE Tutorial Child
Text Domain: fse-tutorial-child

There is no need to enqueue the style.css for block themes unless your style.css includes custom styles. If your style.css includes custom styles, you can enqueue it in the same manner as the classic theme.  Learn how to [enqueue style.css](https://developer.wordpress.org/themes/advanced-topics/child-themes/#3-enqueue-stylesheet). 

If your child theme includes theme.json, it will overwrite its parent’s theme.json

If a child theme includes a PHP template and its parent includes a block template for the same specificity, the child theme’s PHP template will take precedence.

**Comment by TeBenachi** [comment]
Please disregard my above comment. it did not make sense 😔
I agree on updating the existing [child theme](https://developer.wordpress.org/themes/advanced-topics/child-themes/) page. 

I think only [#3 Enqueue stylesheet](https://developer.wordpress.org/themes/advanced-topics/child-themes/#3-enqueue-stylesheet
) section need to be updated with the block theme info. 

**Suggestion:**

```
The final step is to enqueue the parent and child theme stylesheets if needed.

Block themes do not require enqueuing style.css. Style.css are registered 
automatically. If your style.css includes custom styles, follow the 
instruction below.

For classic themes and block themes that include custom styles, the ideal 
way of enqueuing stylesheets is...
```

**Comment by TeBenachi** [comment]
For [Enqueueing Styles and Scripts](https://developer.wordpress.org/themes/advanced-topics/child-themes/#enqueueing-styles-and-scripts) section, [relating to this comment](https://github.com/WordPress/Documentation-Issue-Tracker/issues/108#issuecomment-1012731899), can a child theme for block theme add javascript?

**Comment by carolinan** [comment]
> Block themes do not require enqueuing style.css. Style.css are registered 
> automatically.

I don't think this is accurate?
But the theme only needs to enqueue it if it is used.

**Comment by TeBenachi** [comment]
Thanks for checking. 


**Revised:**

```
The final step is to enqueue the parent and child theme stylesheets if needed.

Block themes only require enqueuing the style.css if it includes custom styles. 
Follow the instruction below. 

For classic themes and block themes that include custom styles, the ideal 
way of enqueuing stylesheets is...


```
